### ISSUE TITLE
Node icon's size in the inspector is not adjusted

### ISSUE THREAD

**Issue by KoBeWi** [godotengine/godot#81759]
### Godot version

4.2 dev4

### System information

Windows 10.0.19045 - Vulkan (Forward+) - dedicated NVIDIA GeForce GTX 1060 (NVIDIA; 30.0.15.1403) - Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz (8 Threads)

### Issue description

When you assign a custom icon to a node and then assign the node to some Node/NodePath property, the inspector will use full icon's size.
![image](https://github.com/godotengine/godot/assets/2223172/2831fdfa-4c62-4249-bbdf-8a9dad15ffc6)

### Steps to reproduce

1.
```GDScript
@icon("res://icon.svg")
extends Node

@export var test: NodePath = "."
```
2. Close and reopen scene

### Minimal reproduction project

N/A

**Comment by Mickeon** [comment]
This has a bit of a side effect, that is at least for me, it makes it easier to tell what classes are "mine" and what aren't. It also most noticeable when you hover over a custom CanvasItem.
But yeah, it shouldn't happen. At least there should be a limit.

**Comment by raulsntos** [comment]
- Fixed by https://github.com/godotengine/godot/pull/75274?

**Comment by KoBeWi** [comment]
Actually it was fixed by #82728, but I forgot about this issue.
